project.md

```diff
@@ -11,41 +11,57 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+_This language is catered towards users of Adobe's animation and video editing platforms who feel that the current language that is used for creating extensions to the platform is either too dificult to learn or that it isn't specific enough for the work they want to do. Currently javascript is the language used to create extensions or manipulate effects for Adobe software._
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+_There is currently a language used for this purpose, but it is a general purpose language, so I think a more specific language would be more appropriate. Since javascript isn't specfic to videos or animations, there is a lot of code required with javascript that I think could be avoided if a language specfic to videos was used._
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+_I've worked with Adobe After Effects, which is one of Adobe's video editing softwares, a lot, so I used javascript to work with its effects. I think Javascript doesn't work super poorly, but I think I could make a language that would make the process of manipulating effects even better._
+
 ### Domain
 
 _Describe the project's domain in five words._
 
+_Graphic, Video, Effects, Manipulation, Extension_
+
 ### Interface (syntax)
 
 _How might the user interact with the language? What does programming look
 like? Why is this the right way to interact with the problem domain?_
 
+_The user isn't going to be writing large chunks of code for the most part, but is going to be using pre written methods to manipulate the effects. To create extensions and effects, the user would have to write more code. The editor is going to be in Adobe After Effects itself. I believe this is the right way to interact with the domain because this is how javascript is currently used with the software._
```

Comment:
Is the idea to make a plugin for Adobe After Effects? I've never used the software, so I'm unfamiliar with how it works and what is easy/difficult to do as a programmer with After Effects.